inst/css/filter-panel.css

```diff
@@ -251,3 +251,22 @@ a.remove_all:hover {
   display: flex;
   justify-content: space-between;
 }
+
+.progress.state-count-container {
+  height: 1.75em;
+  width: 90%;
+  background-color: white;
+  margin: 0px;
+}
+
+.progress-bar.state-count-bar-filtered {
+  background-color: lightblue !important;
+  color: var(--bs-body-color, var(--dark, #333333));
+  overflow: visible;
+  text-align: left;
+  white-space: nowrap;
+}
+
+.progress-bar.state-count-bar-unfiltered {
+  background-color: rgba(173, 216, 230, 0.439) !important;
+}
```

Comment:
@asbates do you know how to apply alpha/opacity to the background only? Opacity affects text and background in the same time which means that it could be barely visible,

![image](https://user-images.githubusercontent.com/6959016/219609727-d394a1ee-bd62-4689-985b-0cbc88d1f4da.png)


```suggestion
.progress-bar.state-count-bar-filtered {
  background-color: var(--bs-primary, rgb(51, 122, 183));
  opacity: 0.8;
  color: var(--bs-body-color, var(--dark, #333333));
  overflow: visible;
  text-align: left;
  white-space: nowrap;
}

.progress-bar.state-count-bar-unfiltered {
  background-color: var(--bs-primary, rgb(51, 122, 183));
  opacity: 0.439;
  border-top-right-radius: var(--bs-border-radius, .375rem);
  border-bottom-right-radius: var(--bs-border-radius, .375rem);
}
```

Reply:
I do not. Bootstrap uses hex for color variables and as of now there isn't a way to add alpha to an existing hex color. (apparently this is in the works). So setting opacity on the whole bars is probably not the way to go. The text looked light to me but I wanted to bring it up just in case it was just me.